File: loianeTraining/javaBasico/Exer Aula 16_17/Ex10.java
import java.util.Scanner;

public class Ex10 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Digite dois numeros");
        var number1 = scan.nextInt();
        var number2 = scan.nextInt();

        var higher = 0;
        var smallest = 0;
        if (number1 > number2) {
            higher = number1;
            smallest = number2;
        } else {
            higher = number2;
            smallest = number1;
        }

        System.out.println("Os numero inteiros entre " + number1 + " e " + number2 + " sao:");
        for (int currentNumber = smallest; currentNumber <= higher; currentNumber++) {
            System.out.print(currentNumber + " ");
        }
        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/d/Ex10.cs
using System;

class Ex10
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite 2 numeros");
        int number1 = int.Parse(Console.ReadLine());
        int number2 = int.Parse(Console.ReadLine());

        int smallerNumber = number1 < number2 ? number1 : number2;
        int greaterNumber = smallerNumber == number1 ? number2 : number1;

        for (int i = smallerNumber + 1; i < greaterNumber; i++)
        {
            Console.Write(i + " ");
        }
    }
}